Dockerfile

```
@@ -1,7 +1,7 @@
 FROM islasgeci/jupyter:ff82
 WORKDIR /workdir
 COPY . .
-RUN Rscript -e "install.packages(c('covr', 'lintr', 'rjson', 'roxygen2', 'testthat'), repos='http://cran.rstudio.com')"
+RUN Rscript -e "install.packages(c('covr', 'lintr', 'rjson', 'roxygen2', 'testthat', 'styler'), repos='http://cran.rstudio.com')"
```

¿Por favor podrías ordenarlos alfabéticamente?:

```suggestion
RUN Rscript -e "install.packages(c('covr', 'lintr', 'rjson', 'roxygen2', 'styler', 'testthat'), repos='http://cran.rstudio.com')"
```